packages/azure-openapi-validator/autorest/src/spectral-plugin-func.ts

```diff
@@ -36,7 +36,8 @@ export async function spectralPluginFunc(initiator: IAutoRestPluginInitiator): P
   }
 
   const resolvedOpenapiType: OpenApiTypes = getOpenapiType(openapiType)
-  const [ruleset, namesOfRulesInStagingOnly]: [Ruleset, string[]] = await getRuleSet(resolvedOpenapiType)
+  // FIXME: This should be set by the CI, not hardcoded
+  const [ruleset, namesOfRulesInStagingOnly]: [Ruleset, string[]] = await getRuleSet(resolvedOpenapiType, true)
```

Comment:
Let's clarify what is this `true` at the end ? Either with appropriately named variable or a comment.